# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../argp-standalone/0001-Makefile.am-add-more-libtoolization.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From e8ea423a775a82ee946ba465f766dca0a5f1a90d Mon Sep 17 00:00:00 2001
From: Christian Wiese <chris@opensde.org>
Date: Thu, 8 Aug 2013 10:49:56 +0200
Subject: [PATCH] Makefile.am: add more libtoolization

---
 Makefile.am |    8 +++++---
 1 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 5b58d46..757feb6 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -21,6 +21,8 @@
 AUTOMAKE_OPTIONS = foreign
 SUBDIRS = . testsuite
 
+lib_LTLIBRARIES = libargp.la
+
 LIBOBJS = @LIBOBJS@
 
 noinst_LIBRARIES = libargp.a
@@ -30,11 +32,11 @@ noinst_HEADERS = argp.h argp-fmtstream.h argp-namefrob.h # argp-comp.h
 EXTRA_DIST = mempcpy.c strchrnul.c strndup.c Versions
 
 # Leaves out argp-fs-xinl.c and argp-xinl.c 
-libargp_a_SOURCES = argp-ba.c argp-eexst.c argp-fmtstream.c \
+libargp_la_SOURCES = argp-ba.c argp-eexst.c argp-fmtstream.c \
 		    argp-help.c argp-parse.c argp-pv.c \
 		    argp-pvh.c
 
-libargp_a_LIBADD = $(LIBOBJS)
+libargp_la_LIBADD = $(LIBOBJS)
 
-argp_test_LDADD = libargp.a
+argp_test_LDADD = libargp.la
 
-- 
1.7.2.3


From 38b1213290277deca769723d595ef7c700db992c Mon Sep 17 00:00:00 2001
From: Christian Wiese <chris@opensde.org>
Date: Thu, 8 Aug 2013 11:51:43 +0200
Subject: [PATCH] more libtoolization (SQUASH)

---
 configure.ac          |    2 ++
 testsuite/Makefile.am |    2 +-
 2 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index 4658839..d819cff 100644
--- a/configure.ac
+++ b/configure.ac
@@ -12,6 +12,8 @@ AM_CONFIG_HEADER(config.h)
 # systems; no problems have been reported with it so far.
 AC_GNU_SOURCE
 	
+LT_INIT
+
 # Checks for programs.
 AC_PROG_CC
 AC_PROG_MAKE_SET
diff --git a/testsuite/Makefile.am b/testsuite/Makefile.am
index 9993541..0e34120 100644
--- a/testsuite/Makefile.am
+++ b/testsuite/Makefile.am
@@ -5,7 +5,7 @@ TS_ALL = $(TS_PROGS) $(TS_SH)
 
 noinst_PROGRAMS = $(TS_PROGS) ex1 ex3 ex4
 
-LDADD = ../libargp.a
+LDADD = ../libargp.la
 
 EXTRA_DIST = $(TS_SH) run-tests
 CLEANFILES = test.out
-- 
1.7.2.3


From e7955d68fbc73049f682e67ac0963ea28266b62d Mon Sep 17 00:00:00 2001
From: Christian Wiese <chris@opensde.org>
Date: Thu, 8 Aug 2013 12:15:30 +0200
Subject: [PATCH] SQUASH

---
 Makefile.am |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 757feb6..466e192 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -25,7 +25,6 @@ lib_LTLIBRARIES = libargp.la
 
 LIBOBJS = @LIBOBJS@
 
-noinst_LIBRARIES = libargp.a
 noinst_PROGRAMS = argp-test
 noinst_HEADERS = argp.h argp-fmtstream.h argp-namefrob.h # argp-comp.h
 
-- 
1.7.2.3


From 0b0b7f32a61769c8c6c7c4656bf92a6457ca2055 Mon Sep 17 00:00:00 2001
From: Christian Wiese <chris@opensde.org>
Date: Thu, 8 Aug 2013 12:20:16 +0200
Subject: [PATCH] SQUASH

---
 Makefile.am |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 466e192..fe8122c 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -23,10 +23,12 @@ SUBDIRS = . testsuite
 
 lib_LTLIBRARIES = libargp.la
 
+include_HEADERS = argp.h
+
 LIBOBJS = @LIBOBJS@
 
 noinst_PROGRAMS = argp-test
-noinst_HEADERS = argp.h argp-fmtstream.h argp-namefrob.h # argp-comp.h
+noinst_HEADERS = argp-fmtstream.h argp-namefrob.h # argp-comp.h
 
 EXTRA_DIST = mempcpy.c strchrnul.c strndup.c Versions
 
-- 
1.7.2.3

